component.json

```diff
@@ -2,7 +2,7 @@
   "title": "SFTP",
   "buildType": "docker",
   "description": "Provides file access and transfer using SSH File Transfer Protocol",
-  "version": "1.4.9",
+  "version": "1.5.0-dev.2",
```

Comment:
```suggestion
  "version": "1.4.10-dev.2",
```

Reply:
New functionality and changes in UI (added passphrase field)

Reply:
I see. my bad